### DISCUSSION TITLE
How to pass a storage class with persistent data to Dify helm chart?

### DISCUSSION THREAD

**Discussion by masoodamunchen** [langgenius/dify#8158] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I am using your dify helm chart to deploy on my EKS cluster. Using ArgoCD manifest as well for CD. How can I pass a storage class to your helm chart (override) so that it uses my own persistent storage class?

**Comment by crazywoola** [comment]
Have you seen these repos.

[Helm Chart by @LeoQuote](https://github.com/douban/charts/tree/master/charts/dify)
[Helm Chart by @BorisPolonsky](https://github.com/BorisPolonsky/dify-helm)
[YAML file by @Winson-030](https://github.com/Winson-030/dify-kubernetes)

**Reply by masoodamunchen** [reply]
@crazywoola Thanks! Helm Chart by @LeoQuote seems interesting as I can see how to inject the external postgres using the backend env override.
